### PR TITLE
feat(input): improve the input configuration 

### LINKED ISSUE #19: find the closed pull request
Could it find the closed pull request which triggers from the head branch and base branch? I want to reopen and reuse the same PR. Thanks! 😁

### LINKED ISSUE #20: use github.token as the default token input
See in reference: [About the `GITHUB_TOKEN` secret](https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret).

### PR DESCRIPTION
**Pull request by yi-Xu-0100**
1. use the `github.token` as the default token input.
2. add more input options for queries.

Close #19
Close #20

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -16,9 +17,23 @@ inputs:
   base:
     description: 'The base branch name of the Pull Request'
     required: false
-  author: 
+  author:
     description: 'The author of the Pull Request'
     required: false
+  state:
+    description: 'The state of the Pull Request'
```
Comment (juliangruber) [inline action.yml:24]: Could you also list here which states are supported?

File: action.yml [hunk 2]
```diff
@@ -16,9 +17,23 @@ inputs:
...
+    required: false
+    default: open
+  sort:
+    description: |
+      What to sort results by. Can be either `created`, `updated`, `popularity`
+      (comment count) or `long-running` (age, filtering by pulls updated in the last month).
+    required: false
+  direction:
+    description: |
+      The direction of the sort. Can be either `asc` or `desc`. Default: `desc` when sort is created or sort is not specified, otherwise `asc`.
```
Comment (juliangruber) [inline action.yml:34]:
```suggestion
    description: 'The direction of the sort. Can be either `asc` or `desc`.'
    default: 'desc'
```

File: index.js [hunk 3]
```diff
@@ -8,10 +8,13 @@ const main = async () => {
...
   const base = core.getInput('base')
   const author = core.getInput('author')
+  const state = core.getInput('state')
+  const sort = core.getInput('sort')
+  const direction = core.getInput('direction')
 
   const query = {
     ...context.repo,
-    state: 'open'
+    state: state
```
Comment (juliangruber) [inline index.js:17]:
```suggestion
    state
```

**Review by juliangruber — CHANGES_REQUESTED** [review]
Looking great, almost there!

File: action.yml (hunk 1 above)
Comment (yi-Xu-0100) [reply action.yml:24]: @juliangruber Sure, I forget it. 😅

Comment (yi-Xu-0100) [inline action.yml:24]:
```suggestion
    description: 'The state of the Pull Request. Can be `open`, `closed`, and `all`.'
```

File: action.yml [hunk 4]
```diff
@@ -16,9 +17,23 @@ inputs:
   base:
     description: 'The base branch name of the Pull Request'
     required: false
-  author: 
+  author:
     description: 'The author of the Pull Request'
     required: false
+  state:
+    description: 'The state of the Pull Request. Can be `open`, `closed`, and `all`.'
```
Comment (yi-Xu-0100) [inline action.yml:24]:
```suggestion
    description: 'The state of the Pull Request. Can be either `open`, `closed` or `all`.'
```

**Comment by juliangruber** [comment]
https://github.com/juliangruber/find-pull-request-action/releases/tag/v1.5.0 🤝

**Pull request merged**